stable/concourse/values.yaml

```diff
@@ -1578,6 +1575,13 @@ secrets:
   ##
   create: true
 
+  ## array of team names and public keys for team external workers
```

Comment:
(nit) The sentence should start with an uppercase letter and end with a period so that we make that consistent throughout the file.